seed_db.sh

```diff
@@ -70,164 +70,230 @@ else
-
-        docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', 3, 'My old gentlemen felt that these words were very important too—’Thou shalt’ and ‘Do thou.’ 
-            And this was the gold from our mining: ‘Thou mayest.’ ‘Thou mayest rule over sin.’ ');"
-
-        docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', 4, 'The old gentlemen smiled and nodded and felt the years were well spent. It brought them out of their
-            Chinese shells too, and right now they are studying Greek.” ');"
-        
-        docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', 5, 'Samuel said, “It’s a fantastic story. And I’ve tried to follow and maybe I’ve missed somewhere. 
-            Why is this word so important?”');"
-        
-        docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', 6, 'Lee’s hand shook as he filled the delicate cups. 
-            He drank his down in one gulp. “Don’t you see?” he cried. ');"
-        
-        docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', 7, '“The American Standard translation orders men to triumph over sin, 
-            and you can call sin ignorance. ');"
-        
-        docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', 8, 'The King James translation makes a promise in ‘Thou shalt,’ 
-            meaning that men will surely triumph over sin. ');"
-        
-        docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', 9, 'But the Hebrew word, the word timshel—‘Thou mayest’—that gives a choice. 
-            It might be the most important word in the world. 
-            That says the way is open. That throws it right back on a man.
-            For if ‘Thou mayest’—it is also true that ‘Thou mayest not.’ Don’t you see?”');"
+        docker exec -it planet-read_db_1 mysql -u root -proot -e "
```

Comment:
nit: the indentation showing up in the editable cells is not great looking or representative of how users will likely write out translations, can we make these strings have no new lines?
![image](https://user-images.githubusercontent.com/15113249/127957175-c0e0b0b2-8bb5-402a-b400-e39f25217421.png)